brvs-rules/src/main/java/iroha/validation/rules/impl/sora/AssetTransferBlockedRule.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package iroha.validation.rules.impl.sora;
+
+import iroha.protocol.Commands.Command;
+import iroha.protocol.Commands.TransferAsset;
+import iroha.protocol.TransactionOuterClass.Transaction;
+import iroha.validation.rules.Rule;
+import iroha.validation.verdict.ValidationResult;
+import java.util.Arrays;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class AssetTransferBlockedRule implements Rule {
+
+  private final Set<String> assetsBlocked;
+
+  public AssetTransferBlockedRule(String assetsBlocked) {
+    this.assetsBlocked = Arrays.stream(assetsBlocked.split(",")).collect(Collectors.toSet());
```

Comment:
There can be only one single asset to block? If so, why not just store this asset instead of set? If no, why constructor allow to add only one asset, but not multiple?

Reply:
I assume a more flexible way so a set can be configured